### DISCUSSION TITLE
Get a Global Environnement

### DISCUSSION THREAD

**Discussion by Jzerbib69** [usebruno/bruno#1590] [Q&A]
Hi, i got a little problem, i start to configure all my env for my collection, but i have same env for all collections so i dont want to push vars on each collections env, how can i do this ??

Example on postman 
![Sans titre](https://github.com/usebruno/bruno/assets/28700975/d64bb29a-ce71-404d-a518-eb979acf7357)

**Comment by paulrudy** [comment]
Here's [one way](https://github.com/usebruno/bruno/issues/115#issuecomment-1873970728)
